### PR TITLE
STORM-912: Support SSL on Logviewer

### PR DESCRIPTION
**Pull request by knusbaum**
Backporting STORM-912 (#604) fix to 0.10.x branch.

### CODE REVIEW

**Comment by revans2** [comment]
+1

**Comment by d2r** [comment]
+1 The travis-ci error was strange.  It was looking for calcite version 0.9.2-incubating-SNAPSHOT.  I built on OSX using the travis-ci command, and it built and passed all tests.

**Comment by HeartSaVioR** [comment]
#609 resolves storm-hive calcite SNAPSHOT dependency issue, but I didn't backport it to 0.10.x-branch since it seems to be minor for users, but it is critical issue for travis-ci build.
When we're willing to do pull request against 0.10.x-branch, we may be better to backport #609 to 0.10.x-branch, too.

**Comment by d2r** [comment]
> #609 resolves storm-hive calcite SNAPSHOT

Thanks, @HeartSaVioR I was really confused when I saw it.

**Comment by HeartSaVioR** [comment]
Thinking it once more, we should maintain 0.10.x-branch for a fairly long time, so travis-ci should be fixed to future PRs.
I addressed backporting #609 to #718.

File: storm-core/src/clj/backtype/storm/ui/core.clj [hunk 1]
```diff
@@ -951,17 +958,17 @@
   (GET "/api/v1/topology/summary" [:as {:keys [cookies servlet-request]} & m]
        (assert-authorized-user servlet-request "getClusterInfo")
        (json-response (all-topologies-summary) (:callback m)))
-  (GET  "/api/v1/topology/:id" [:as {:keys [cookies servlet-request]} id & m]
+  (GET  "/api/v1/topology/:id" [:as {:keys [cookies servlet-request schema]} id & m]
```
Comment (HeartSaVioR) [inline storm-core/src/clj/backtype/storm/ui/core.clj]: @knusbaum 
I don't know about compojure / ring, and I can't find related docs, but core.clj from master branch uses different key name.
Actually master branch uses both schema and scheme.

- route which uses schema as key: https://github.com/apache/storm/blob/master/storm-core/src/clj/backtype/storm/ui/core.clj#L997
- route which uses scheme as key: https://github.com/apache/storm/blob/master/storm-core/src/clj/backtype/storm/ui/core.clj#L1004

Which one is correct? Or both things are correct?

**Comment by HeartSaVioR** [comment]
@knusbaum Ping.

**Comment by knusbaum** [comment]
@HeartSaVioR Sorry, it took me a while to get back to this. 

The keyword is scheme, not schema. I've fixed it here, and I'll create a new pull request to master to fix it there, too.

**Comment by HeartSaVioR** [comment]
Great! +1

**Pull request merged**